src/aero_kernel/src/fs/inode.rs

```diff
@@ -309,6 +326,28 @@ impl DirEntry {
         })
     }
 
+    pub fn from_inode_custom(
+        parent: DirCacheItem,
+        name: String,
+        inode: Arc<dyn INodeInterface>,
```

Comment:
Unused argument :^)

Reply:
Fixed in 89405b11de214d018f8dceeee09e9f354ed59755